docs/reference/sql-openapi/explicit-include.md

```diff
@@ -0,0 +1,14 @@
+# Explicitly including entities
+
+`@platformatic/sql-openapi` allows for specifying entities to be included. **Note**:
+using this method will ignore any unspecified entities in the schema.
+
+To include entities:
+
+```javascript
+app.register(require('@platformatic/sql-openapi'), {
+  include: {
+    category: true
+  }
+})
+```
```

Comment:
This docs is about entities, but it should be about tables (considering the implementation).

Reply:
Good point. Adjustments made.

Reply:
Actually I misread the code.

So, sql-mapper deals with tables, while OpenAPI deals with Entities.